docs/index.md

```diff
@@ -224,6 +226,57 @@ resource "grafana_oncall_escalation" "example_notify_step" {
 - `tls_key` (String) Client TLS key (file path or literal value) to use to authenticate to the Grafana server. May alternatively be set via the `GRAFANA_TLS_KEY` environment variable.
 - `url` (String) The root URL of a Grafana server. May alternatively be set via the `GRAFANA_URL` environment variable.
 
+### Managing Connections
+
+#### Obtaining Connections access token
+
+Before using the Terraform Provider to manage Grafana Connections resources, such as metrics endpoint scrape jobs, you need to create an access policy token on the Grafana Cloud Portal. This token is used to authenticate the provider to the Grafana Connections API.
+[These docs](https://grafana.com/docs/grafana-cloud/account-management/authentication-and-permissions/access-policies/authorize-services/#create-an-access-policy-for-a-stack) will guide you on how to create
+an access policy. The required permissions, or scopes, are `integration-management:read`, `integration-management:write` and `stacks:read`.
+
+Also, by default the Access Policies UI will not show those scopes, to find name you need to use the `Add Scope` textbox, as shown in the following image:
+
+<img src="https://grafana.com/media/docs/grafana-cloud/aws/cloud-provider-terraform-access-policy-creation.png" width="700"/>
```

Comment:
Should this use a separate screenshot instead of reusing the cloud provider one? Just thinking that the name of the access token is called `cloud-provider-terraform`. We may not want to encourage people to use the same access token for both, in case our needs ever diverge permissions-wise.

Reply:
Yeah makes sense, this came up on the PR for it https://github.com/grafana/terraform-provider-grafana/pull/1861#discussion_r1819165332

Do you have guidance on where to host?

Reply:
I'm not sure where @thepalbi found `https://grafana.com/media/docs/grafana-cloud/aws/` and how that got setup. If there isn't a similar folder for Connections, maybe good to create one. Otherwise I've seen searching in the Grafana org that plenty of screenshots exist just in `https://grafana.com/media/docs/grafana-cloud/` https://github.com/search?q=org%3Agrafana+%22media%2Fdocs%2Fgrafana-cloud%2F%22&type=code

Reply:
Updated in 791c7be017d73b48b4ef3663b3a87c4baaf9ff90